package-lock.json

```diff
@@ -4,17 +4,44 @@
 	"lockfileVersion": 1,
 	"requires": true,
 	"dependencies": {
+		"@newrelic/koa": {
+			"version": "1.0.5",
+			"resolved": "https://registry.npmjs.org/@newrelic/koa/-/koa-1.0.5.tgz",
+			"integrity":
+				"sha512-1zTojq9gW2mi0YblGrS86gCyL56+gbCn6o2+1UJJL3pFmBgp8IAMzZ93PkHHtdrbL3BnVMBrD2Q2WR32FbhIAg==",
+			"requires": {
+				"methods": "1.1.2"
+			}
+		},
+		"@newrelic/native-metrics": {
+			"version": "3.1.0",
+			"resolved":
+				"https://registry.npmjs.org/@newrelic/native-metrics/-/native-metrics-3.1.0.tgz",
+			"integrity":
+				"sha512-45OhFKiRT5L+hBzAkDkR5UTuHWIUNMjwlMlu65Y8HmyrTPtgUMKoqMmHERRX4aRqb/EEaW3oWYYLOpLnB6fiHg==",
+			"optional": true,
+			"requires": {
+				"nan": "2.10.0"
+			}
+		},
 		"@sinonjs/formatio": {
 			"version": "2.0.0",
 			"resolved":
-				"https://registry.npmjs.org/@sinonjs/formatio/-/formatio-2.0.0.tgz",
+				"http://registry.npmjs.org/@sinonjs/formatio/-/formatio-2.0.0.tgz",
```

Comment:
do we know why its changged to http?

Reply:
Don't really know how that happens. May be new `npm` did this.